packages/jumpstarter-driver-flashers/oci_bundles/build_bundle.sh

```
@@ -0,0 +1,23 @@
+set -e
+
+FLASHER_OCI_CONTAINER="${1:-quay.io/jumpstarter-dev/jumpstarter-flasher-test:latest}"
+BUNDLE_FILES=${2:-"./test/"}
+
+echo "Building and pushing ${FLASHER_OCI_CONTAINER}"
+
+set -x
+
+cd "${BUNDLE_FILES}"
+DATA_FILES=
+for file in $(find ./data -type f); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)